kratos/modeler/create_entities_from_geometries_modeler.cpp

```diff
@@ -0,0 +1,168 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/parallel_environment.h"
+#include "utilities/entities_utilities.h"
+#include "utilities/variable_utils.h"
+#include "create_entities_from_geometries_modeler.h"
+
+namespace Kratos
+{
+
+namespace {
+
+/**
+ * @brief Create entities from geometries.
+ * @param EntityName the name of the entity
+ * @param rModelPart the model part
+ */
+template <class TEntitiesContainerType, class TEntityIdentifier>
+void CreateEntitiesFromGeometries(
+    const TEntityIdentifier& rEntityIdentifier,
+    ModelPart& rModelPart
+    )
+{
+    // Create the entities container and allocate space
+    TEntitiesContainerType entities_to_add;
+    entities_to_add.reserve(rModelPart.NumberOfGeometries());
+
+    // Get current max element id
+    using SizeType = std::size_t;
```

Comment:
Move outside the function because its used again in 81, 91, 14, 158. Its not clear if its getting the definition from here or from another include

Reply:
Agree. Actually I don't know why I defined it in here instead of doing so in the header.

Reply:
Done.